src/rust/bitbox02-rust/src/hww/api.rs

```diff
@@ -94,11 +94,17 @@ fn can_call(request: &Request) -> bool {
         Uninitialized,
         // Seeded (password defined, seed created/loaded).
         Seeded,
-        // Initialized (seed backuped up on SD card).
-        Initialized,
+        // InitializedAndLocked (seed backuped up on SD card, keystore locked).
+        InitializedAndLocked,
+        // InitializedAndUnlocked (seed backuped up on SD card, keystore unlocked).
+        InitializedAndUnlocked,
```

Comment:
State enum comment could be updated to reflect the changes

Reply:
Done